straxen/scada.py

```diff
@@ -237,7 +237,7 @@ def _query_single_parameter(self,
         if query_type_lab:
             # In the lab case we get interpolated data without nans so the df can be set
             # accordingly.
-            seconds = np.arange(start, end + 1, 10**9, every_nth_value)
+            seconds = np.arange(start, end + 1, 10**9 * every_nth_value)
```

Comment:
whoops how did this go undetected before..

Reply:
It was not used yet and it is still not really implemented as it does not work from the WebAPI side. Independent of what I query I always get a 1 min separation between values.